controllers/argocd/role_test.go

```diff
@@ -64,7 +64,7 @@ func TestReconcileArgoCD_reconcileRole_for_new_namespace(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, expectedNumberOfRoles, len(dexRoles))
 	assert.Equal(t, expectedRoleNamespace, dexRoles[0].ObjectMeta.Namespace)
-	
+
```

Comment:
please revert. no real change in this file.

Reply:
Ack